### ISSUE TITLE
Social description is pulling from the wrong place

### ISSUE THREAD

**Issue by dawatorbert** [semperfiwebdesign/all-in-one-seo-pack#675]
This is the homepage of https://yakmybody.com. It seems to be pulling from the regular SEO title/desc for the page rather than the social one.

![screen shot 2016-12-06 at 8 02 20 pm](https://cloud.githubusercontent.com/assets/18234064/20950655/0373c490-bbef-11e6-950c-980cdab38153.png)
![screen shot 2016-12-06 at 8 02 01 pm](https://cloud.githubusercontent.com/assets/18234064/20950656/03777ba8-bbef-11e6-8cb6-1081cc82de4a.png)
![screen shot 2016-12-06 at 8 01 50 pm](https://cloud.githubusercontent.com/assets/18234064/20950657/038348b6-bbef-11e6-8edb-436a54d8706e.png)

**Comment by dawatorbert** [comment]
The same thing was happening with the social image. It was pulling from the default setting, not what was set in the (static) homepage. Deselecting "Use AIOSEO Title and Description" in the Social module fixes it so that the title, desc, and image can be set there.

This isn't intuitive, and we should figure out something better...

![screen shot 2016-12-06 at 8 50 06 pm](https://cloud.githubusercontent.com/assets/18234064/20951749/a99310c8-bbf5-11e6-9959-097c0e641ae7.png)

**Comment by dawatorbert** [comment]
Actually, it looks like the title and desc are coming from the site title and tagline, regardless of what's put in the static home page's seo title/desc, social title/desc, or general or social settings title/desc... It seems to ignore all 4 of those places.

**Comment by wpsmort** [comment]
You set the Open Graph title, description and image by going to All in One SEO > Social Meta > Home Page Settings.  No need to open an issue for this in All in One SEO Pack as we have an issue open for it and know about this.

**Comment by michaeltorbert** [comment]
@wpsmort That isn't what was happening here. https://github.com/semperfiwebdesign/all-in-one-seo-pack/issues/675#issuecomment-265333796 It's pulling the title and description from the site title and tagline in WP general settings.

**Comment by wpsmort** [comment]
@michaeltorbert I did not see this behavior when I checked the live site and I cannot replicate this on my dev site.  

For the SEO Title and OG Title: If you have the Use Static Front Page Instead option enabled then we use the page title as the SEO title which in the case of my dev site is Home.  However, if you don't have the Use Static Front Page Instead option enabled then we use the Site Title.

For the SEO Description and OG Description: We don't output anything for the SEO description but we do output the Tagline for the OG Description.  

We know there are a number of issues here with the way this code works and there are way too many options that affect this.  As discussed previously, this needs to be it's own project with some discussion on what options we remove, what we leave and how fall back and backwards compatibility is handled.
